packages/eden/src/links/internal/batched-data-loader.ts

```diff
@@ -62,11 +62,15 @@ export function batchedDataLoader<TKey, TValue>(loader: BatchLoader<TKey, TValue
   const groupItems = (items: BatchItem<TKey, TValue>[]) => {
     const groupedItems: BatchItem<TKey, TValue>[][] = [[]]
 
-    let i = 0
+    let index = 0
```

Comment:
I feel like this isn't any more helpful. `i` and `index` both denote an arbitrary marker, and they're ambiguous about the context. I think it would be most helpful with a comment, regardless of the variable name.

```ts
/** Index of the current batch item being handled. */
let index
```